api/v1alpha08/kogitoserverlessplatform_types.go

```diff
@@ -86,8 +87,39 @@ type KogitoServerlessPlatformSpec struct {
 	BuildPlatform api.PlatformBuildSpec `json:"platform,omitempty"`
 	// Configuration list of configuration properties to be attached to all the Workflow built from this Platform
 	Configuration ConfigurationSpec `json:"configuration,omitempty"`
+	// Trait list of traits properties attached to all the Workflow built from this Platform
+	Traits Traits `json:"traits,omitempty"`
 }
 
+type Traits struct {
+	// The configuration of Service trait
+	Service *trait.ServiceTrait `property:"service" json:"service,omitempty"`
+}
+
+// A TraitSpec contains the configuration of a trait
+// Deprecated: superceded by each Trait type, left for backward compatibility.
```

Comment:
I think that since this is deprecated and not used at the moment we can remove it.
The same for all the other structs that are marked as deprecated.